integrations/git_test.go

```diff
@@ -216,7 +216,10 @@ func rawTest(t *testing.T, ctx *APITestContext, little, big, littleLFS, bigLFS s
 			req = NewRequest(t, "GET", path.Join("/", username, reponame, "/raw/branch/master/", littleLFS))
 			resp = session.MakeRequest(t, req, http.StatusOK)
 			assert.NotEqual(t, littleSize, resp.Body.Len())
-			assert.Contains(t, resp.Body.String(), models.LFSMetaFileIdentifier)
+			assert.LessOrEqual(t, resp.Body.Len(), 1024)
+			if resp.Body.Len() != littleSize && resp.Body.Len() <= 1024 {
```

Comment:
This condition is already asserted above, right?
Can't we check for the exact expected length?
Are there tests for small and large files?

Reply:
The reason for the changes being the way they are is because of the contains below this line.

If the resp.Body does not contain the expected value it will be emitted whole into the test response.

This is bad and completely unreadable - we therefore need to skip doing that test in that case.